scripts/exglobal_marine_analysis_checkpoint.py

```diff
@@ -25,4 +25,7 @@
     MarineAnl.execute('soca_incpostproc')
 
     # Compute the observation space statistics
-    MarineAnl.execute('soca_diag_stats')
+    try:
+        MarineAnl.execute('soca_diag_stats')
+    except Exception as e:
+        logger.warning(f"WARNING: Execution of soca_diag_stat application failed: {e}")
```

Comment:
`execute` raises a Workflow exception, and the program will terminate. I wonder if the `try-except` is relevant or needed here?